src/DonorProfiles/resources/js/app/components/rest-subscription-table/index.js

```diff
@@ -8,16 +8,16 @@ const RESTSubscriptionTable = () => {
 		<Table
 			header={
 				<Fragment>
-					<div className="give-donor-profile-table__column">
+					<div className="give-donor-dashboard-table__column">
 						Subscription
```

Comment:
@henryholtgeerts Can we localize these strings?

Reply:
Ah good catch! This components is actually meant to be removed (replaced by SubscriptionTable instead). I've now removed this logic, ref: https://github.com/impress-org/givewp/pull/5708/commits/ba86799ecdac13b18d894dffc003e970b25b23d5